<launch>
	<param name="/camera_prefix" type="string" value="/" />
		
	<node pkg="visp_auto_tracker" type="visp_auto_tracker" name="visp_auto_tracker" output="screen">
		  <param name="model_path" type="string" value="$(find visp_auto_tracker)/models" />
		  <param name="model_name" type="string" value="pattern" />
		  <remap from="/visp_auto_tracker/camera_info" to="/camera_info"/>
		  <remap from="/visp_auto_tracker/image_raw" to="/image_rect"/>
		
		<remap from="/visp_auto_tracker/object_position_covariance" to="/object_position_covariance"  />
		<remap from="/visp_auto_tracker/object_position" to="/object_position"  />
	</node>


	  <node pkg="camera1394" type="camera1394_node" name="my_camera1394_node" args="_video_mode:=640x480_rgb8" >
	    <param name="camera_info_url" value="package://visp_auto_tracker/models/calibration.ini" type="string" />
	    <param name="frame_id" value="/map" type="string" />
	    <remap from="/camera/image_raw" to="/image_rect"/>
	    <remap from="/camera/camera_info" to="/camera_info"/>
	  </node>
  
	<node pkg="visp_tracker" type="viewer" name="tracker_mbt_viewer" args="_camera_prefix:=/">
		<param name="model_path" value="$(find visp_auto_tracker)/data/pattern.wrl" />
  	</node>
	
	
</launch>